lib/options/uefi-setup-menu.robot

```diff
@@ -86,7 +86,7 @@ Measure Coldboot Time
     Log To Console    \n
     # Do one more iteration than requested, as we may hit first boot which is always longer.
     FOR    ${index}    IN RANGE    0    ${iterations}+1
-        Power Cycle On    power_button=${TRUE}
+        Power Cycle On
```

Comment:
@SebastianCzapla Please see and rebase on top of it instead using similar approach: https://github.com/Dasharo/open-source-firmware-validation/pull/615

Adding this argument for one platform, and removing for another (without testing the results on different platforms at the same time) will not lead to anything good.

The updated `Power Cycle On` version from my PR should help here.